### ISSUE TITLE
Issue with Wiris filter and Moodle 4.2

### ISSUE THREAD

**Issue by Nenuial** [wiris/moodle-filter_wiris#121]
When testing the upgrade to Moodle 4.2 I run into a serious issue with the Wiris Filter plugin :
```php
uncaught exception: require_once(/var/www/html/lib/editor/tinymce/lib.php): Failed to open stream: No such file or directory (errno: 2) in /var/www/html/filter/wiris/classes/pluginwrapper.php at line #160require_once(/var/www/html/lib/editor/tinymce/lib.php): Failed to open stream: No such file or directory

in file: /var/www/html/filter/wiris/classes/pluginwrapper.php line 160
#0 /var/www/html/filter/wiris/classes/pluginwrapper.php(160): _hx_error_handler()
#1 /var/www/html/filter/wiris/classes/pluginwrapper.php(160): require_once()
#2 /var/www/html/filter/wiris/classes/configurationupdater.php(51): filter_wiris_pluginwrapper::get_wiris_plugin()
#3 /var/www/html/filter/wiris/classes/pluginwrapper.php(85): filter_wiris_configurationupdater->__construct()
#4 /var/www/html/filter/wiris/classes/pluginwrapper.php(54): filter_wiris_pluginwrapper->init()
#5 /var/www/html/filter/wiris/settings.php(35): filter_wiris_pluginwrapper->__construct()
#6 /var/www/html/lib/classes/plugininfo/filter.php(138): include('...')
#7 /var/www/html/admin/settings/plugins.php(273): core\plugininfo\filter->load_settings()
#8 /var/www/html/lib/adminlib.php(8709): require('...')
#9 /var/www/html/admin/upgradesettings.php(22): admin_get_root()
#10 {main}
```

This makes the Admin panel inusable. 

I guess it might have something to do with the legacy MCE editor which was removed in Moodle 4.2. `lib/editor/tinymce` is no longer present in 4.2, only `lib/editor/tiny`.

**Comment by jguillen-at-wiris** [comment]
Hi Nenuial!

Thanks for reporting your issue.
We will review it and bring you feedback ASAP.

**Comment by Nenuial** [comment]
Hi,
Yes by admin panel I mean the "site administration page". The error shown above appears when trying to access it.

We have update our Moodle test version from 4.1 to 4.2 and that's when the problem first appeared. Uninstalling filter-wiris solves the problem, but that's not an option going forward since our users depend on Wiris quizzes.

**Comment by Nenuial** [comment]
Just as an input, I tried this morning to install the plugin with the changes in #122 and it solves the problem on my side. I can now use MathType in the new TinyMCE 6.

**Comment by carla-at-wiris** [comment]
Hi Nenuial! Indeed, that was the issue. The current moodle version requires using Tiny6 paths. We have forced the path of our requirements for Moodle versions 4.2 and above to look for that path by default.

We also released a specific MathType plugin for TinyMCE 6, you can check it [in here](https://moodle.org/plugins/tiny_wiris). We'll update our documentation in the future!

Notice that this fix will be available in our next release `v8.2.7`.

Since you already fixed it already, I will close this issue. Please, feel free to reopen it if necessary.

**Comment by Nenuial** [comment]
Thanks, I have indeed installed the plugin for TinyMCE6 and it does work for us.
Thank you for solving this !
